servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TerminalSignalConsumer.java

```diff
@@ -40,4 +42,14 @@ default void onError(Throwable throwable) {
      */
     default void onCancel() {
     }
+
+    /**
+     * Creates a new {@link TerminalSignalConsumer} from {@link Runnable}.
```

Comment:
```suggestion
     * Adapts the passed {@link Runnable} to a {@link TerminalSignalConsumer} such that {@link Runnable#run()} 
     * is invoked for each call to {@link TerminalSignalConsumer#onError()}, {@link TerminalSignalConsumer#onComplete()} and {@link TerminalSignalConsumer#onCancel()}.
```

Reply:
This should be `{@link TerminalSignalConsumer#onError(Throwable)}`